src/main/java/org/opensearch/agent/tools/RAGTool.java

```diff
@@ -193,8 +187,30 @@ public <T> void run(Map<String, String> parameters, ActionListener<T> listener)
             log.error("Failed to search index.", e);
             listener.onFailure(e);
         });
-        vectorDBTool.run(Map.of(VectorDBTool.INPUT_FIELD, embeddingInput), actionListener);
 
+        Map<String, Object> params = new HashMap<>();
+        params.put(VectorDBTool.INDEX_FIELD, this.index);
```

Comment:
How about using AbstractRetrieverTool.INDEX_FIELD? Same to below lines.

Reply:
No longer needed